soc/arm/silabs_exx32/efr32mg12p/Kconfig.soc

```diff
@@ -13,7 +13,7 @@ config SOC_EFR32MG12P
 	bool "SOC_EFR32MG12P"
 	select HAS_SILABS_GECKO
 	select HAS_CMU
-	select HAS_EMU
+	select SOC_GECKO_EMU
```

Comment:
This should not be necessary any more. We should be able to remove this line (I did not test it).